source/server/admin/stats_handler.cc

```diff
@@ -104,6 +104,14 @@ Http::Code StatsHandler::handlerStats(absl::string_view url,
     }
   }
 
+  for (const Stats::CounterGroupSharedPtr& counter : server_.stats().counterGroups()) {
+    if (shouldShowMetric(*counter, used_only, regex)) {
+      for (size_t i = 0; i < counter->maxEntries(); ++i) {
+        all_stats.emplace(absl::StrCat(counter->name(), counter->nameSuffix(i)), counter->value(i));
```

Comment:
should we add a `"."` in between the suffix name and the counter-group name?

this makes me wonder also whether there's a test that hits this, where you might have noticed the lack of `"."`